tst/EnergyPlus/unit/OutputFiles.unit.cc

```diff
@@ -174,6 +174,11 @@ TEST_F(EnergyPlusFixture, OutputFiles_Expected_Formatting_Tests)
     //    EXPECT_EQ(format("{:N}", 999.9), "           999.9    ");
     EXPECT_EQ(format("{:N}", 61677162.0987027), "61677162.0987027    ");
 
+    // Tested with E+ 9.3.0
+    EXPECT_EQ(format("{:N}", 0.03576388888888889), "          0.3576E-01");
+    EXPECT_EQ(format("{:.3N}", 0.03576388888888889), "0.358E-01");
+    EXPECT_EQ(format("{:.4N}", 0.03576388888888889), "0.3576E-01");
+
```

Comment:
New format test

Reply:
It says in the comments "Tested with E+ 9.3.0", does that mean `NumberOut` below will be "0.3576E-01"?

```c++
ObjexxFCL::gio::Fmt fmtLD("*");
std::string NumberOut;
ObjexxFCL::gio::write(NumberOut, fmtLD) << repVal;
strip_trailing_zeros(strip(NumberOut));
```